examples/common/scripts/vtgate-up.sh

```diff
@@ -39,6 +39,7 @@ vtgate \
   --tablet_types_to_wait PRIMARY,REPLICA \
   --service_map 'grpc-vtgateservice' \
   --pid_file $VTDATAROOT/tmp/vtgate.pid \
+  --enable_buffer \
```

Comment:
This merely makes it easier for users to test buffering related behavior in the local examples moving forward.